Features2/tests/features/userRegistration.feature

```diff
@@ -0,0 +1,31 @@
+Feature:User registration
+    As a user
+    I want to register my account 
+    so as  I can have valid credentials to login to use the Contact Fox
+     
+    Background:The user is in Registration page 
+        Given a user has navigated to the loginpage
+        And the user has clicked Register button
+        And the user has been navigated to the  Registration page
+
+
+        Scenario:To register account with valid email and  same password and confirm password
+            When the user enters the name with "name" and email with "valid email" and password with "password" and confirm password with "password"
```

Comment:
You have used some actual data in the example table in line no. 19. Use similar values here.